metals/src/main/scala/scala/meta/internal/builds/BloopInstall.scala

```diff
@@ -132,24 +132,28 @@ final class BloopInstall(
           scribe.info(s"skipping build import with status '${result.name}'")
           Future.successful(result)
         case _ =>
-          scribe.debug("Awaiting user response...")
-          for {
-            userResponse <- requestImport(
-              buildTools,
-              buildTool,
-              languageClient,
-              digest,
-            )
-            installResult <- {
-              if (userResponse.isYes) {
-                runUnconditionally(buildTool, isImportInProcess)
-              } else {
-                // Don't spam the user with requests during rapid build changes.
-                notification.dismiss(2, TimeUnit.MINUTES)
-                Future.successful(WorkspaceLoadedStatus.Rejected)
+          if (userConfig().automaticImportBuild) {
```

Comment:
Should we only import automatically at first or also later on?

If we want to always import automatically after changes to build files I would suggest starting it after the focused document changes, as otherwise it will be imported on each saved change, maybe missing some of the changes while the import is underway,.

Reply:
Not sure I follow, you're suggesting we don't wait for the document (presumably a `build.sbt`) to be saved, but instead just be edited? Or are you saying we should only run when the doc is saved and has also received an update?

Reply:
The problem is if someone has auto save on or saves maniacally like me :sweat_smile: In that case the import will be started at first save and later changes will be ignored until the first import finishes. This might not be an issue if someone controls it well, but would be good to avoid surprises here anyway. So instead we could do only import if the build file is not currently focused, which would indicate the user stopped editing the build definition.

An easier solution would be to do automatic import only at the very first import.

Reply:
Ah I follow you now. I feel like only doing things on unfocus is probably also liable to be surprising. I think just scoping this to the initial import as you suggest is probably the most straightforward solution.  I guess in practice that means reverting the changes to `Indexer` whilst retaining these ones to `BloopInstall` if I've understood correctly?

Reply:
I double checked and I think you are right, otherwise it might be really hard to get right.